config/schemas/config.yaml

```diff
@@ -14,6 +14,220 @@ $defs:
     thing:
       $ref: "#/$defs/thing"
     ```
+  affinity:
+    $comment: Real k8s resources might have existing schemas that could be reused as-is
+    additionalProperties:
+      type: object
+    default:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: node-role.kubernetes.io/control-plane
+                  operator: Exists
```

Comment:
This feels like an example and not a suitable default.

Reply:
Yeah, this is from `.fluentd.forwarder` in `config/config/wc-config.yaml`, I'll move it to `.examples[0]` instead.

Reply:
Done in d5f91280